test/db/userChallenge.test.js

```diff
@@ -0,0 +1,22 @@
+require('@babel/polyfill');
+const { UserChallenge } = require('../../server/db/models');
```

Comment:
I changed the name of the model so that it would be easier for us to use
`UserChallenge.js` => `Userchallenge.js`

```suggestion
const { Userchallenge } = require('../../server/db/models');
```